1-mandatory/2-quotegenerator/quotes.js

```diff
@@ -490,3 +490,16 @@ const quotes = [
     author: "Zig Ziglar",
   },
 ];
+const quoteP = document.querySelector("#quote");
+const authorP = document.querySelector("#author");
+const quoteBtn = document.querySelector("#new-quote");
+let autoPlayInterval;
```

Comment:
This variable is unused and should be removed

Reply:
Do you mean I should use `document.querySelector("#quote").textContent = quote;` directly?